.php_cs.dist

```diff
@@ -73,7 +71,9 @@ return Config::create()
         '@Symfony:risky' => true,
         '@PHP71Migration' => true,
         '@PHP71Migration:risky' => true,
+        '@PHP73Migration' => true,
```

Comment:
Our minimum php version is 7.2 right now i think, will this cause any issues?

Reply:
Indeed :/